frontend/public/components/droppable-edit-yaml.tsx

```diff
@@ -94,11 +98,18 @@ export const DroppableEditYAML = withDragDropContext(
   },
 );
 
+type EditYAMLProps = {
+  obj: string;
+  fileUpload: string;
+  error: string;
+  onDrop: (item: any, monitor: any) => void;
```

Comment:
can't we use `DropTargetMonitor` type ? 
```suggestion
  onDrop: (item: any, monitor: DropTargetMonitor) => void;
```

Reply:
:-)